index.html

```diff
@@ -63,15 +63,40 @@
                     bounds: {
                         start: Date.now() - THIRTY_MINUTES,
                         end: Date.now()
-                    }
+                    },
+                    presets: [
+                        {
+                            label: 'Last Day',
+                            bounds: {
+                                start: Date.now() - 1000 * 60 * 60 * 24,
+                                end: Date.now()
+                            }
+                        },
+                        {
+                            label: 'Last 2 hours',
+                            bounds: {
+                                start: Date.now() - 1000 * 60 * 60 * 2,
+                                end: Date.now()
+                            }
+                        },
+                        {
+                            label: 'Last hour',
+                            bounds: {
+                                start: Date.now() - 1000 * 60 * 60,
+                                end: Date.now()
+                            }
+                        }
+                    ],
+                    // maximum entries to retain in conductor history
+                    records: 10
```

Comment:
Can we have the records preset be set as default, so when timesystems dont have this option set, it uses the default 10 vs failing completely

Reply:
Yes. See this line in ConductorHistory.vue
`const records = configuration ? configuration.records : DEFAULT_RECORDS;`